tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/reindexing/ElasticsearchReindexingTests.java

```diff
@@ -61,7 +64,7 @@ public class ElasticsearchReindexingTests extends AbstractTestNGSpringContextTes
 
     private UserModel testUser;
     private SiteModel testSite;
-    private RestHighLevelClient elasticClient;
+    private OpenSearchClient elasticClient;
```

Comment:
I see the test failures in the build. Kindly check that?

Reply:
All Tests Suites are green in Build except "All AMPs Tests". 
All AMPs Tests are failing due to duplicate libraries. 
Damian told that Azure Connector needs to be fixed and released. He will take care of that.